scripts/docker-compose-viz.bash

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Creates a graph out of a docker-compose.yml
```

Comment:
This is great, we can automatically setup a task to build and update the container graph!

Reply:
Yes. See ``make auto-doc`` in main ``Makefile``